src/index.jsx

```diff
@@ -20,25 +20,31 @@ import {
   LibraryEditPage,
   LibraryListPage,
   LibraryCreatePage,
-  StudioHeader,
   LibraryAccessPage,
   LibraryAuthoringPage,
+  StudioHeaderWrapper,
 } from './library-authoring';
 import './index.scss';
 import './assets/favicon.ico';
 
 mergeConfig({
+  LIB_AUTHORING_BASE_URL: process.env.BASE_URL,
   STUDIO_BASE_URL: process.env.STUDIO_BASE_URL,
+  LOGO_URL: process.env.LOGO_TRADEMARK_URL,
   BLOCKSTORE_COLLECTION_UUID: process.env.BLOCKSTORE_COLLECTION_UUID,
   SECURE_ORIGIN_XBLOCK_BOOTSTRAP_HTML_URL: process.env.SECURE_ORIGIN_XBLOCK_BOOTSTRAP_HTML_URL,
 });
 
 subscribe(APP_READY, () => {
   ReactDOM.render(
     <AppProvider store={store}>
-      <div className="wrapper">
-        <StudioHeader />
-        <main>
+        {/* todo: look for a better way to do this, using these routes in
```

Comment:
my main issue with this implementation is having multiple route switches in the `index.jsx` file here

i did add it in https://github.com/openedx/frontend-app-library-authoring/commit/f9349871fa7dac7bea9a8580d8df3d7bdbb5b19e to clear the state (since `loadingStatus` would stay `loaded` when navigating back to the list page)

i was hoping i'd have some better ideas by now, but nothing is coming to mind

Reply:
For what it's worth, it seems to me like a legitimate use of `Switch`, and a reasonable place to put it in. 🤷🏼

Reply:
sounds good, i'll remove the todo